ddtrace/_trace/trace_handlers.py

```diff
@@ -646,9 +646,10 @@ def _on_botocore_kinesis_update_record(ctx, stream, data_obj: Dict, record, inje
         HTTPPropagator.inject(ctx[ctx["call_key"]].context, data_obj["_datadog"])
 
 
-def _on_botocore_update_messages(ctx, span, _, trace_data, __, message=None):
+def _on_botocore_update_messages(ctx, span, _, trace_data, __, message=None) -> Dict:
     context = span.context if span else ctx[ctx["call_key"]].context
-    HTTPPropagator.inject(context, trace_data)
+    HTTPPropagator.inject(context, trace_data["_datadog"])
```

Comment:
Does this negatively affect any of the other integrations that trigger the `_on_botocore_update_messages` function?

Reply:
Distressing that unit tests didn't catch this. Fixed!